platform/cri-o/conmon.patch

```diff
@@ -0,0 +1,307 @@
+diff --git a/BUILD.bazel b/BUILD.bazel
+new file mode 100644
+index 0000000..760175f
+--- /dev/null
++++ b/BUILD.bazel
+@@ -0,0 +1,68 @@
++# TODO: include -std=c99 in C builds
++
++# genrule(
++#     name = "config.h-genrule",
++#     outs = [":conf/config.h"],
++#     tools = ["//cmd/crio-config"],
++#     cmd = """
++#         REL="$$(realpath "$(location //cmd/crio-config)")"
++#         cd \"$(@D)\"
++#         "$${REL}"
++#     """,
++# )
```

Comment:
Why are these blocks commented out instead of removed?

Reply:
Oh whoops, I missed that completely. Thanks for catching that!